src/LightGraphs.jl

```diff
@@ -138,8 +135,8 @@ include("core.jl")
             include("persistence/graphml.jl")
             include("persistence/net.jl")
             include("persistence/jld.jl")
-        include("randgraphs.jl")
-        include("generators.jl")
+        include("generators/staticgraphs.jl")
+            include("generators/randgraphs.jl")
```

Comment:
to have the docs, where there is a single page, graph generators,documenting static graphs and randgraphs,  coherent with the folder structure